.github/workflows/scheduled.yml

```diff
@@ -527,3 +635,21 @@ jobs:
           path: |
             /tmp/aggregate_fuzzer_repro
             /tmp/server.log
+
+  surface-signature-errors:
```

Comment:
Does this need to be separate workflow, cant we do it at the end of the failing workflow ?

Reply:
This is meant to surface the issue quickly (e.g. in case it wasn't intentional), if we add it to the bias fuzzer it will show up  an hour later. The job takes a few seconds so it's not like it costs anything ^^